bin/wasm-node/javascript/tsconfig-cjs.json

```
@@ -0,0 +1,11 @@
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    "lib": [
+      "ES2015"
+    ],
+    "target": "ES2015",
```

```suggestion
      "es2015"
    ],
    "target": "es2015",
```

That's really nitpicky, but let's keep it consistent